.github/CODEOWNERS

```diff
@@ -15,6 +15,12 @@
 # Module: logger
 /packages/logger/ @SAP/ux-tools-logger
 
+# Module: project-access
+/packages/project-access/ @SAP/ux-tools-project-access
```

Comment:
Could we have one team for both modules? We have started doing this also for other modules where the same persons will be responsible for multiple modules.

I would recommend `@SAP/ux-tools-project-access` for both

Reply:
I really liked the fact that we had a 1:1 mapping of team to module. Many things can change and to me 1:1 seems to be the most flexible, clean, and future proof choice. 
However, we can also start with one team for both and split it as needed. 
Changed it for now.